client/components/tinymce/plugins/simple-payments/style.scss

```diff
@@ -52,12 +60,20 @@
 	flex: auto;
 	overflow-y: auto;
 
+	@include breakpoint( ">480px" ) {
+		padding: 24px;
+	}
+
 	@include breakpoint( ">660px" ) {
 		display: flex;
+
+		.upload-image {
+			margin-right: 25px;
+		}
 	}
 
 	.upload-image {
-		margin-right: 25px;
+		margin-bottom: 20px;
```

Comment:
I noticed that most paddings and margins in Calypso are multiples of 8px (16px, 24px, ...), forming kind of a grid system for spacings. I'm not sure how much it's a part of the Calypso design language -- it'd be nice to adhere to it if it is.